mozilla-release/mobile/android/base/java/org/mozilla/gecko/tabs/TabsPanelThumbnailView.java

```diff
@@ -23,8 +24,8 @@
 
     /* Cliqz start */
     private Path clipPath;
-
     private float[] roundedRectRadii;
+    private RectF rectF = new RectF();
```

Comment:
its better to be final, and initialize it in the Third Constructor then use it in onDraw function.